Yoast/YoastSEO.js
# Add Spanish full text tests

## Summary

This PR can be summarized in the following changelog entry:

* Adds full text tests for Swedish

## Test instructions

This PR can be tested by following these steps:

* Run the tests.
* Verify the tests pass.
* Run the example webpack: cd examples/webpack&&yarn&&yarn start.
* Verify the analyze spam in the example works

Fixes #2044
